api/migrations/202011051807132-addMissingMineGuid.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+const ObjectId = require('mongoose').Types.ObjectId;
+
+let dbm;
+let type;
+let seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+}
+
+exports.up = async function(db) {
+  console.log('**** Updating Fording River and KSM Records****');
+  const mClient = await db.connection.connect(db.connectionString, { native_parser: true });
+
+  try {
+    const nrpti = await mClient.collection('nrpti');
+    
+    // Updates any records that have an EPIC project ID set to Fording River and sets their mineGuid to the correct Core mine which
+    // maps them to the Fording River project in NRPTI.
+    await nrpti.updateMany({ 
+      _epicProjectId: ObjectId('588511a6aaecd9001b823734'),
+      _schemaName: { $in: [
+        'CertificateBCMI',
+        'OrderBCMI',
+        'InspectionBCMI',
+        'CertificateAmendmentBCMI',
+        'ManagementPlanBCMI'
+      ]},
+      mineGuid: null
+    },
+    {  $set: { mineGuid: '739e90ee-807c-4287-91a3-7099352e1ffa' } });
+
+     // Updates any records that have an EPIC project ID set to KSM and sets their mineGuid to the correct Core mine which
+    // maps them to the KSM Mine project in NRPTI.
+    await nrpti.updateMany({ 
+      _epicProjectId: ObjectId('58851156aaecd9001b81e652'),
+      _schemaName: { $in: [
+        'CertificateBCMI',
+        'OrderBCMI',
+        'InspectionBCMI',
+        'CertificateAmendmentBCMI',
+        'ManagementPlanBCMI'
+      ]},
+      mineGuid: null
```

Comment:
Do records where `mineGuid: ""` also require fixing?

Reply:
I could add it as a safety, but the particular records we are targeting will have it set to `null` and there are none with a blank string.

Reply:
Ah I see the data on the live db has been altered.  I do have a bunch of records in my copy of the post data load DB with just empty string mineGuid.  Might still be a good idea to add blank string check.

Reply:
Updated to check for a bank string too.